### DISCUSSION TITLE
Can't drag multiple items in Boards view

### DISCUSSION THREAD

**Discussion by drone1** [community/community#11863] [Projects and Issues]
I can select multiple items with ``CTRL+LEFT CLICK`` or ``SHIFT+LEFT CLICK`` but dragging between columns only moves one object at a time.

**Comment by MohamedBengezi** [comment]
Any comments on this from someone at GitHub?

**Comment by drone1** [comment]
GitHub does not interact with their community and doesn’t listen to what
their community wants.

I think the conclusion has to be pretty clear at this point, as someone who
follows multiple threads for basic, high-demand features: GitHub/Microsoft
simply does not care.
